### DISCUSSION TITLE
Deploying my app on cloudways hosting

### DISCUSSION THREAD

**Discussion by hassam-saeed** [vikejs/vike#1417] [Discussion]
I am deploying my app to cloudways that are already deployed on Vercel. So I have just cloned my GitHub repo and started the server and it comes with this error. 

Now my question is how do I copy the vercel config files to another hosting?

Just started npm run dev and hit the main route and it gives me an error like this:
<img width="1290" alt="Screenshot 2024-01-08 at 6 59 12 PM" src="https://github.com/vikejs/vike/assets/51261422/2b1ae309-bc45-4975-8278-b13587cfca79">

[GitHub Repo](https://github.com/DevAxonOfficial/SSR-Frontend)

**Comment by brillout** [comment]
https://brillout.github.io/rules/

**Reply by hassam-saeed** [reply]
I am facing an issue and you are continuously closing my discussion. This is not fair @brillout
